### PR TITLE
[Feature: #162727562] Tag articles

### PR DESCRIPTION
**Pull request by habibaudu**
#### What does this PR do?
- Tag Articles
#### Description of Task to be completed?
- [x] Change tags column data type in the article models to  Array
- [x] Tag Articles
#### How should this be manually tested?
- Fire up postman
- Url: [Here](http://localhost:4000/api/v1/articles/tag)
- Enter `tags` and `title`
- Click Send

#### What are the relevant pivotal tracker stories?
[#162727562](https://www.pivotaltracker.com/story/show/162727562)

### CODE REVIEW

File: models/article.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false
     },
     tags: {
-      type: DataTypes.STRING
+      type: DataTypes.ARRAY(DataTypes.TEXT)
```
Comment (oreoluwade) [inline models/article.js]: Replace `DataTypes.TEXT` with `DataTypes.STRING`

File: migrations/009-create-articles-update-tag.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface
+    .changeColumn('Articles', 'tags', {
+      type: `${Sequelize.ARRAY(Sequelize.TEXT)}USING CAST("tags" as 
+      ${Sequelize.ARRAY(Sequelize.TEXT)})`
```
Comment (oreoluwade) [inline migrations/009-create-articles-update-tag.js]: Replace `Sequelize.TEXT` with `Sequelize.STRING`

**Review by oreoluwade — CHANGES_REQUESTED** [review]

**Comment by oreoluwade** [comment]
- Remove the space that is line 68 in the article controllers file.

- The coverage has dropped. Found that line 54 is not tested. This means that there's no test for when tags have already been added before and a new tag is added. So you need to add one more test spec.

File: models/article.js (hunk 1 above)
Comment (habibaudu) [reply models/article.js]: ok

**Comment by habibaudu** [comment]
> * Remove the space that is line 68 in the article controllers file.
> * The coverage has dropped. Found that line 54 is not tested. This means that there's no test for when tags have already been added before and a new tag is added. So you need to add one more test spec.

ok

**Review by oreoluwade — APPROVED** [review]

**Pull request merged**